packages/terra-icon/scripts/src/README.md

```diff
@@ -1,10 +1 @@
-# Update all SVG icons
-
-```
-npm run migrate-cerner-one-icons
-```
-* npm run compilescripts
-* npm run migrate-csv
-* npm run migrate-svg
-* npm run generate-icon
-* npm run generate-example
+Each script has its own readme. Instructions on when and how to use these scripts are in the terra-icons readme.
```

Comment:
```suggestion
Each script has its own readme. Instructions on when and how to use these scripts are in the [terra-icons readme](insert link to readme).
```

Reply:
Fixed in 9115d408060a5416be60684ef6246049fedc0023